apps/checkout/src/lib/utils/environment.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-restricted-globals */
+
+export type EnvVar =
+  | "checkoutAppUrl"
+  | "apiUrl"
+  | "devCheckoutToken"
+  | "configAppUrl";
+
+export interface EnvVars
+  extends Omit<Record<EnvVar, string>, "devCheckoutToken"> {
+  devCheckoutToken?: string;
+}
+
+export const getEnvVars = (): EnvVars => {
```

Comment:
This does not need to be a function, you could either use `ENV_VARS` object or just export each one of them individually as a variable (this approach will save you some lines in typing things)

Reply:
I'd rather keep it as a typed object, so when used it's obvious which ones are needed for the app to work and which ones are optional (rather than manually doing `as string` when used in e.g api requests). But totally right, no need for a function here